tools/cross_runner/kernels/conv_nchw_fp16.cpp

```diff
@@ -242,6 +256,7 @@ extern "C" _GENX_MAIN_ void convolution_nchw_nondpas(
 	const uint32_t activation_type = constants[9];
 	const uint32_t activation_alpha = constants[10];
 	const uint32_t activation_beta = constants[11];
+	const uint32_t output_layout = constants[12];
```

Comment:
Please rename it to be a bit more meaningful, for example: `output_layout_is_nhwc`